pytest_examples/test_class.py

```diff
@@ -2,82 +2,101 @@
 
 from pytest_examples.class_to_test import ClassToTest
 
+from freezegun import freeze_time
+
+import datetime
+
 @pytest.fixture
 def any_fit():
     return ClassToTest()
 
+
+@freeze_time("2021-12-09")
```

Comment:
this is using freezegun directly, which we also do in our tests. 

But instead I would like to you to use [pytest-freezgun](https://pypi.org/project/pytest-freezegun/) which works as a fixture.`def test_frozen_date(freezer):`